packages/ui-app/src/Params/Params.css

```diff
@@ -2,6 +2,10 @@
 /* This software may be modified and distributed under the terms
 /* of the ISC license. See the LICENSE file for details. */
 
+.ui--Param-Address {
+  font-family: 'Courier New', Courier, monospace;
```

Comment:
Just make this `monospace` consistent with what we have in the rest of the UI, e.g. https://github.com/polkadot-js/apps/blob/master/packages/ui-app/src/styles/components.css#L54